data/tr2/ship/data/scripts/monastry.lua

```diff
@@ -0,0 +1,6 @@
+trx.events.on_level_init(function(level)
+  trx.creatures.add_ally(trx.catalog.Objects.O_MONK_1)
+  trx.creatures.add_ally(trx.catalog.Objects.O_MONK_2)
+  trx.creatures.add_ally_target(trx.catalog.Objects.O_BANDIT_1)
+  trx.creatures.add_ally_target(trx.catalog.Objects.O_BANDIT_2)
```

Comment:
Ideally it would be cool to create a two-way relationship, so we can make bandits shoot both monks AND crows, for example, even though crows are not allies.